pom.xml

```diff
@@ -299,6 +300,7 @@
                         <exclude>${exclude.tests}</exclude>
                     </excludes>
                     <systemPropertyVariables>
+                        <PALISADE_REST_CONFIG_PATH>example/example-model/src/main/resources/configRest.json</PALISADE_REST_CONFIG_PATH>
```

Comment:
Could this be an environment variable instead of a System property?

Reply:
made change and committed to branch